sanic/config.py

```diff
@@ -201,7 +201,11 @@ def load_environment_vars(self):
         for k, v in os.environ.items():
             if k.startswith(SANIC_PREFIX):
                 _, config_key = k.split(SANIC_PREFIX, 1)
-                if v.isdigit():
-                    self[config_key] = int(v)
+                # This is a float or an int
```

Comment:
@yunstanford oh really, I've heard it was the other way... oh well, I'm done changing this back and forth :) if someone wants to modify it further we're happy to have another PR.